src/templates/styles.css

```
@@ -58,17 +58,23 @@ body {
   transform: translate(-50%, -50%);
 }
 
+#loading-text {
```

```suggestion
#loading-text {
```
We should check this together